### ISSUE TITLE
How does the Bandwidth feature fit with kernel versions earlier than 5.1

### ISSUE THREAD

**Issue by Charlottell** [cilium/cilium#18639]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

BPF bandwidth manager needs kernel 5.1 or newer. Disabling the feature.

### Cilium Version

1.11.0

### Kernel Version

Linux master1 4.19.0-arm64-server #3017 SMP Thu Nov 12 19:54:04 CST 2020 aarch64 GNU/Linux

### Kubernetes Version

1.20

### Relevant log output

```shell
BPF bandwidth manager needs kernel 5.1 or newer. Disabling the feature.
```

### Anything else?

When Cilium is deployed, I want to use the Bandwidth function and get this prompt, "BPF bandwidth manager needs kernel 5.1 or newer".
Because I want to adapt to different kernel versions, why not support kernel versions lower than 5.1? Is there any missing configuration for a version below 5.1 that makes it unusable? What configuration do I need to add if I want to adapt different kernel versions? Please give me some advice. Thanks in advance.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Charlottell** [comment]
I looked at the source code and find the information: “We at least need 5.1 kernel for native TCP EDT integration and writable queue_mapping that we use. Below helper is available for 5.1 kernels and onwards”

**Comment by aanm** [comment]
@Charlottell I assume your question is answered and we can close the issue? Thank you
